src/screens/Assets/collectibles/CollectibleListItem.js

```diff
@@ -45,7 +45,7 @@ function CollectibleListItem({ title, iconUrl, onPress, width }: Props) {
       <IconContainer width={width}>
         {!!iconUrl && (
           <>
-            {isSvg && <SvgCssUri uri={iconUrl} width="100%" height="100%" />}
+            {isSvg && <SvgImage source={{ uri: iconUrl }} style={[{ width: 100, height: 100 }]} />}
```

Comment:
```suggestion
            {isSvg && <SvgImage source={{ uri: iconUrl }} style={[{ width: 100, height: 100 }]} />}
```

Hm, it was 100% dimensions so I guess you want that to be "100%"?